server/services/payment-service.js

```diff
@@ -9,5 +17,15 @@ module.exports = {
     }
     console.log('creating payment')
     await paymentRepository.create(calculation)
+  },
+  getAll: async function () {
+    const payments = await paymentRepository.getAll()
+    const returnValue = payments.map(paymentMapper)
+    return returnValue
```

Comment:
You could return the result of `payments.map` rather than assigning it to a variable, then returning it. This would be consistent with the approach taken in the `schedule-service`

Reply:
This must be a common error pattern resulting from the addition and removal of `console.log(...)` statements.